public/api/definitions/_schemas.yml

```diff
@@ -837,6 +837,7 @@ sampleAttributeMap:
       - type: 'null'
       - type: number
       - type: boolean
+      - type: hash
```

Comment:
should be `type: object` I think

Reply:
or maybe even remove this specification since it includes every possible type.

See also: https://github.com/seek4science/seek/blob/main/public/api/definitions/_schemas.yml#L840-L842

maybe could re-use that?

Reply:
consolidated into a single `attributeMap` definition